test/integration/serializers/sample.json

```diff
@@ -0,0 +1,30 @@
+[
+  "category", {"class": ["api"], "title": "My API", "description": "An API description."}, {}, [
+    ["category", {"class": ["resourceGroup"], "title": "My Group", "description": "This is a group of resources"}, {}, [
+      ["copy", {}, {"contentType": "text/plain"}, "Extra text"],
+      ["resource", {"title": "Frob", "description": "A frob does something."}, {
+        "href": "/frobs/{id}",
+        "hrefVariables": ["hrefVariables", {}, {}, [
+            ["string", {"name": "id", "description": "Unique identifier"}, {"typeAttributes": ["required"]}, ""]
+          ]]
+        }, [
+        ["dataStructure", {}, {}, [
+          ["string", {"name": "id"}, {}, null],
```

Comment:
Note this has been changed in refract base spec see https://github.com/refractproject/refract-spec/pull/27 (as a rule of thumb – there is no longer `name` in meta